.github/workflows/workflow_lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
+
+on:
+  workflow_call:
+    inputs:
+      ci_type:
+        type: string
+        default: 'pr'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-ft
```

Comment:
please make it unique. This value is for finetune. You can use "${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-lint".